ext/hash/hash.stub.php

```diff
@@ -8,13 +8,17 @@
  */
 const HASH_HMAC = UNKNOWN;
 
-/** @refcount 1 */
+/**
+ * @compile-time-eval
```

Comment:
like `crc32` hash should be CTEable

currently the following code:
```
$h0 = hash("murmur3a", "Two hashes meet in a bar.", options: ["seed" => 2345]);
```

is causing a segfault with opcache/JIT:
```
========DIFF========
     95855f9be0db784a5c37e878c4a4dcee
002+ Segmentation fault (core dumped)
003+ 
004+ Termsig=11
002- 95855f9be0db784a5c37e878c4a4dcee
003- f64c9eb40287fa686575163893e283b2
004- f64c9eb40287fa686575163893e283b2
005- 7f7ec59b
006- 7f7ec59b
========DONE========
FAIL Hash: MurmurHash3 seed [ext/hash/tests/murmurhash3_seed.phpt] 
```

any idea how to fix it?

Reply:
tracked in #10801